konflux-ci/rbac/core/konflux-admin-user-actions.yaml

```diff
@@ -206,60 +145,10 @@ rules:
     resources:
       - serviceaccounts/token
   - verbs:
-      - create
-    apiGroups:
-      - ''
-    resources:
-      - pods/exec
```

Comment:
What's the reason for dropping this?

Reply:
we are not allowing this downstream. not sure if we need it here. cc: @gbenhaim